src/admin/WL_Metabox/WL_Metabox_Field_date.php

```diff
@@ -2,14 +2,37 @@
 
 
 class WL_Metabox_Field_date extends WL_Metabox_Field {
+	
+	/*
+	 * Attribute to distinguish between date formats, inferred from the schema property export type
+	 * 
+	 * @since 3.2.0
+	 */
+	private $date_format;
+	
+	/*
+	 * Boolean flag to decide if the calendar should include time or not
+	 * 
+	 * @since 3.2.0
+	 */
```

Comment:
add here the details about the variable:

    * @access private
    * @var boolean $timepicker A boolean flag.